README.md

```diff
@@ -1,284 +1,59 @@
-
-## PyPGStac
-PGStac includes a Python utility for bulk data loading and managing migrations.
-
-PyPGStac is available on PyPI
-```
-pip install pypgstac
-```
+**Source Code**: <a href="https://github.com/stac-utils/pgstac" target="_blank">https://github.com/stac-utils/pgstac</a>
 
-By default, PyPGStac does not install the `psycopg` dependency. If you want the database driver installed, use:
-
-```
-pip install pypgstac[psycopg]
-```
-
-Or can be built locally
-```
-git clone https://github.com/stac-utils/pgstac
-cd pgstac/pypgstac
-pip install .
-```
-
-```
-pypgstac --help
-Usage: pypgstac [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  --install-completion  Install completion for the current shell.
-  --show-completion     Show completion for the current shell, to copy it or
-                        customize the installation.
-
-  --help                Show this message and exit.
-
-Commands:
-  initversion  Get initial version.
-  load         Load STAC data into a pgstac database.
-  migrate      Migrate a pgstac database.
-  pgready      Wait for a pgstac database to accept connections.
-  version      Get version from a pgstac database.
-```
-
-PyPGStac will get the database connection settings from the **standard PG environment variables**:
-
-- PGHOST=0.0.0.0
-- PGPORT=5432
-- PGUSER=username
-- PGDATABASE=postgis
-- PGPASSWORD=asupersecretpassword
-
-It can also take a DSN database url "postgresql://..." via the **--dsn** flag.
-
-### Migrations
-PyPGStac has a utility to help apply migrations to an existing PGStac instance to bring it up to date.
-
-There are two types of migrations:
- - **Base migrations** install PGStac into a database with no current PGStac installation. These migrations follow the file pattern `"pgstac.[version].sql"`
- - **Incremental migrations** are used to move PGStac from one version to the next. These migrations follow the file pattern `"pgstac.[version].[fromversion].sql"`
+---
 
-Migrations are stored in ```pypgstac/pypgstac/migration`s``` and are distributed with the PyPGStac package.
+Lorem ipsum: **PgSTAC** is a set of SQL function and schema to build highly performant database for Spatio-Temporal Asset Catalog (STAC). The project also provide **pypgstac** python module to help with the database migration and documents ingestion (collections and items).
 
-### Running Migrations
-PyPGStac has a utility for checking the version of an existing PGStac database and applying the appropriate migrations in the correct order. It can also be used to setup a database from scratch.
+PgSTAC Documentation: https://stac-utils.github.io/pgstac/pgstac
 
-To create an initial PGStac database or bring an existing one up to date, check you have the pypgstac version installed you want to migrate to and run:
-```
-pypgstac migrate
-```
+pyPgSTAC Documentation: https://stac-utils.github.io/pgstac/pypgstac
```

Comment:
I moved pgstac and pypgstac documentation to specific pages, but now we need to write a quick intro here.

What PgSTAC is, what it does, why people might need it.
Also introduce pypgstac.